setup.js

```diff
@@ -39,13 +39,12 @@ const teardown = () => {
 };
 
 
-teardown()
```

Comment:
This was the only use of the `teardown` function. If we don't do it here, then we'll need to do something manual if we want to clear out the database or if the schema changes (maybe run a migration). Not sure if we need teardown, but if we don't, then the function itself can be removed.